### DISCUSSION TITLE
The media directory cant be accessed in production build (Not Found)

### DISCUSSION THREAD

**Discussion by tiolierobin** [saleor/saleor#12952] [Deployment]
Hi ! finally I already up the production docker setup but I found issue here. somehow the image media product directory cant be accessed when I am running the production instance.

The errors page shown: The requested resource was not found on this server.
<img width="818" alt="Screen Shot 2023-05-27 at 09 24 41" src="https://github.com/saleor/saleor/assets/48957838/bccf03ab-696e-4289-8265-3a85b19723c2">

here the command I run:
`docker run --env ALLOWED_HOSTS=[api.mydomain.com](http://api.mydomain.com/) --env ALLOWED_CLIENT_HOSTS=mydomain.com --env DATABASE_URL=postgres://saleoradmin:mypassword@myIp:port/saleordb --env SECRET_KEY=mysecretkey --env CELERY_BROKER_URL=redis://myIp/1 --env MEDIA_URL=https://api.mydomain.com/media/ --env DEBUG=False --env RSA_PRIVATE_KEY=$(cat private-key.pem) -v ~/saleor-shared-folder:/app/media -p 8000:8000 saleorecommerce`

when I run this on debug mode, it accessable. is there anything that I missed in the configuration?
info: the celery worker is up

here the image url I get:
`https://api.mydomain.com/media/thumbnails/products/Screen_Shot_2023-03-28_at_20.57.42_e27650cc_thumbnail_4096.png`

Thank you !

**Comment by genox** [comment]
Still the same today. DEBUG=False causes the media directory to cause a 404 error.
